src/context/WalletProvider/SelectModal.tsx

```diff
@@ -136,24 +142,40 @@ export const SelectModal = () => {
             ))
           }
         </Grid>
-        <Flex direction={flexDirProp} mt={2} justifyContent='center' alignItems='center'>
-          <Text
-            mb={mbProp}
-            color='text.subtle'
-            translation={walletInfo?.name ? 'common.or' : 'walletProvider.selectModal.footer'}
-          />
-          <Button
-            variant='link'
+        {!walletInfo?.name && (
```

Comment:
don't think we should have this so we have the option available either on no wallet paired, or switch wallet flow (with wallet paired)

Reply:
The previous flow was for `Don't have a wallet?` to be shown if there's no wallet connected, not sure if it makes sense now.

Probably a question for @reallybeard

Reply:
Yah i don't think we need this anymore, should always show I think.

Reply:
https://github.com/shapeshift/web/pull/6061/commits/16eb26955f2f0a7966d49816b172050cd852c471